static/scripts/helpers/AlertMessageController.js

```diff
@@ -23,111 +23,49 @@ class AlertMessageController {
 	buildMessage(message) {
 		const icon = getIconTag(message.status);
 
-		// show only 150 charckters of message
-		let messageText;
-		if (message.text.length > 113) {
-			messageText = `${message.text.substring(0, 113)}...`;
-		} else {
-			messageText = message.text;
-		}
-
-		// if message includes url
-		let url = '';
-		if (message.url) {
-			url = `
-				<a href="${message.url}" rel="noopener" target="_blank" style="float: right;">
-					${message.url.replace(/(^\w+:|^)\/\//, '')}
-				</a>
-				`;
-		}
-
 		const item = document.createElement('div');
 		if (this.loggedin) {
 			item.className = 'alert-item';
 			item.innerHTML = `
-			<div class="alert-date text-nowrap text-muted">
-					${datetime.fromNow(message.timestamp)}
-			</div>
 			<div class="alert-title">${icon} ${message.title}</div>
 			${message.text}
-			${url}
+			<br>
+			<div class="alert-date text-nowrap text-muted" style="float: left;">Updated: ${datetime.fromNow(message.timestamp)} <span style="margin-right: 2px;"></span>|</div>
```

Comment:
Avoid using float and magic number if  it's possible

Reply:
In this situation im working with the same code styles that is place it on legacy client in current file to keep the consistency.